src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs

```diff
@@ -103,6 +103,11 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
         [ValidateNotNullOrEmpty]
         public bool? AllowClassicOperations { get; set; }
 
+        [Parameter(
```

Comment:
You should ensure the new parameter is covered in one of your tests

Reply:
I've updated an existing test to test this flag.